### PR TITLE
installation doesn't create migration table when there are no migrations to play

### PR DESCRIPTION
**Pull request by rfauglas**
**Description (for Contributor and Core Developer)**

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | Todo
| Added legacy Behats               | Todo
| Added acceptance tests            | Todo
| Added integration tests           | Todo
| Changelog updated                 | Todo
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | Todo
| Migration script                  | -
| Tech Doc                          | -

Tested with docker-compose run -u www-data --rm php php bin/console --env=test pim:installer:db and docker-compose run -u www-data --rm php php bin/console --pim:installer:db

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: src/Akeneo/Platform/Bundle/InstallerBundle/Command/DatabaseCommand.php [hunk 1]
```diff
@@ -356,4 +362,13 @@ protected function launchCommands()
 
         return $this;
     }
+
+    /**
```
Comment (nmarniesse) [inline src/Akeneo/Platform/Bundle/InstallerBundle/Command/DatabaseCommand.php:366]: You can skip docblock as the argument and the return of the method are type hinted

**Review by nmarniesse — APPROVED** [review]

File: src/Akeneo/Platform/Bundle/InstallerBundle/Command/DatabaseCommand.php [hunk 2]
```diff
@@ -316,6 +317,11 @@ private function setLatestKnownMigration(InputInterface $input): void
     {
         $latestMigration = $this->getLatestMigration($input);
 
+        if ($this->hasNoMigrationToExecute($latestMigration)) {
```
Comment (BitOne) [inline src/Akeneo/Platform/Bundle/InstallerBundle/Command/DatabaseCommand.php:320]: It's not normal that there's no migration to execute...

File: src/Akeneo/Platform/Bundle/InstallerBundle/Command/DatabaseCommand.php [hunk 3]
```diff
@@ -31,6 +31,7 @@
  */
 class DatabaseCommand extends Command
 {
+    const NO_MIGRATION_OUTPUT = "0\n";
```
Comment (jmleroux) [inline src/Akeneo/Platform/Bundle/InstallerBundle/Command/DatabaseCommand.php:34]: Too hacky for me and mostly we don't explain why there is no migration

**Review by jmleroux — CHANGES_REQUESTED** [review]
Maybe explain in the description why there can be no migration?

**Pull request closed without merging**